builtin/game/chatcommands.lua

```diff
@@ -84,6 +84,15 @@ core.register_chatcommand("me", {
 	end,
 })
 
+core.register_chatcommand("admin", {
+	params = "<action>",
+    privs = {shout = true},
```

Comment:
C&P error again, this only sends to the player who called it so no privs are needed.  Remove the line.